spyder_env_manager/tests/test_plugin.py

```diff
@@ -7,3 +7,262 @@
 """
 Spyder Env Manager plugin tests.
 """
+# Standard library imports
+import logging
+from pathlib import Path
+from unittest.mock import Mock
+
+# Third-party imports
+import pytest
+from qtpy.QtCore import QTimer
+from qtpy.QtWidgets import QMainWindow
+
+# Spyder imports
+from spyder.config.manager import CONF
+
+# Local imports
+from spyder_env_manager.spyder.config import CONF_DEFAULTS
+from spyder_env_manager.spyder.plugin import SpyderEnvManager
+from spyder_env_manager.spyder.widgets.main_widget import (
+    SpyderEnvManagerWidget,
+    SpyderEnvManagerWidgetActions,
+)
+from spyder_env_manager.spyder.widgets.helper_widgets import (
+    CustomParametersDialog,
+)
+
+# Constants
+LONG_OPERATION_TIMEOUT = 600000
```

Comment:
Why does this need to be so long (i.e. 10 min)? Is there a problem with the plugin? Or you just needed to set a value like this to make the tests pass?

Reply:
I think I put that time just to be sure that the tests didn't fail due to a timeout (and instead from actual errors) while I was checking the tests structure locally. I think running all the tests takes around 4 min and the slow ones like 1 min each. Should I change this to 1 min ?

Reply:
Yeah, 1 min or 2 min to be safe would be ok.